packages/appstore/src/assets/svgs/wallets/image-types.tsx

```diff
@@ -0,0 +1,10 @@
+import { WalletsImagesList } from './wallets-image';
+
+export type TWalletsImagesListKey = keyof typeof WalletsImagesList;
+export type TImageTestID = `dt_${TWalletsImagesListKey}`;
+
+export type WalletsImageProps<T> = {
+    image: T;
+    className?: string;
+    width?: number;
+};
```

Comment:
can we move these three types to wallets-image.tsx?
we are importing them inside each other